ophyd/tests/test_quadem.py

```diff
@@ -1,67 +1,36 @@
 import logging
 import pytest
 
-from ophyd import QuadEM, Component as Cpt, Signal
+from ophyd import QuadEM, Kind
+from ophyd.sim import (make_fake_device, clear_fake_device)
 from ophyd.areadetector.plugins import ImagePlugin, StatsPlugin
-from ophyd.ophydobj import Kind
-from .conftest import using_fake_epics_pv
 
 
 logger = logging.getLogger(__name__)
 
 
 @pytest.fixture(scope='function')
-@using_fake_epics_pv
 def quadem():
-    class FakeStats(StatsPlugin):
-        plugin_type = Cpt(Signal, value=StatsPlugin._plugin_type)
-        nd_array_port = Cpt(Signal, value='NSLS_EM')
-
-    class FakeImage(ImagePlugin):
-        plugin_type = Cpt(Signal, value=ImagePlugin._plugin_type)
-        nd_array_port = Cpt(Signal, value='NSLS_EM')
-
-    class FakeQuadEM(QuadEM):
-        image = Cpt(FakeImage, 'image1:')
-        current1 = Cpt(FakeStats, 'Current1:')
-        current2 = Cpt(FakeStats, 'Current2:')
-        current3 = Cpt(FakeStats, 'Current3:')
-        current4 = Cpt(FakeStats, 'Current4:')
-        sum_all = Cpt(FakeStats, 'SumAll:')
-
+    FakeQuadEM = make_fake_device(QuadEM)
     em = FakeQuadEM('quadem:', name='quadem')
+    clear_fake_device(em)
 
-    ''' Beware: Ugly Hack below
```

Comment:
It makes me happy to see lines removed on a comment that says "beware: ugly hack"